README.md

```diff
@@ -1,24 +1,69 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app  allowS you to make them public so anyone can access them.
 
-Things you may want to cover:
+![](./Screenshot%20from%202022-07-02%2007-51-17.png)
 
-* Ruby version
+## EDR diagram
 
-* System dependencies
+![](./recipe_erd_2_members.png)
 
```

Comment:
- [ ] Please make sure to deploy your app and attach the live link in the README file besides the presentation video. 😄

Reply:
@MahmoudBakr23  Thanks for your feedback but I've gone through the project requirements and I haven't been able to see the above requirement. Please let me know if I missed something. Thank you.